src/views/Dissolutions/DefineDissolution.vue

```diff
@@ -59,12 +59,24 @@ import { EntityFilterMixin, EnumMixin } from '@/mixins'
   }
 })
 export default class DefineDissolution extends Mixins(EntityFilterMixin, EnumMixin) {
-  // @Getter
+  // Global getters
   @Getter getBusinessLegalName!: string
   @Getter getDissolutionStatementStep!: DissolutionStatementIF
   @Getter getShowErrors!: boolean
   @Getter isTypeCoop: boolean
 
+  // Global actions
+  @Action setIgnoreChanges!: ActionBindingIF
+
+  /** Called when component is created. */
+  private created (): void {
+    // ignore data changes until page has loaded
+    this.setIgnoreChanges(true)
+    Vue.nextTick(() => {
+      this.setIgnoreChanges(false)
+    })
```

Comment:
This was needed to ignore the initial changes on load. The test for this is:
1. load the filing
2. reload the filing, and watch for no "you have unsaved changes" alert